halconfig/settings.js

```diff
@@ -221,6 +221,7 @@ window.spinnakerSettings = {
     },
   },
   pubsubProviders: ['google'], // TODO(joonlim): Add amazon once it is confirmed that amazon pub/sub works.
+  plugins: ['%plugins%'],
```

Comment:
This needs to match the other templates in the file
```suggestion
  plugins: {%plugins%},
```